docs/python_docs/python/tutorials/deploy/inference/image_classification_jetson.md

```diff
@@ -0,0 +1,130 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# Image Classication using pretrained ResNet-50 model on Jetson module
```

Comment:
Is this for ANY Jetson model? or specifically to Jetson Nano?

Reply:
I have built this wheel for all Jetson SMs - that covers for all Jetson modules - TX1, Nano, TX2, Xavier NX and Xavier AGX. I have currently tested the wheel on Jetson Nano and Xavier AGX and it works on both of them.

Reply:
This tutorial has been tested on the two Jetson modules - Nano and Xavier AGX. I'll specify the device names to be more specific.